action/template_test.go

```diff
@@ -19,30 +21,45 @@ func TestTemplate(t *testing.T) {
 	defer func() { log.Stdout = o }()
 
 	// TOML
-	Template("", tpl, val)
+	Template("", tpl, val, false)
 	if out.String() != "Hello World!\n" {
 		t.Errorf("Expected Hello World!, got %q", out.String())
 	}
 
+	// force
+	err := Template(tpl, val, "", false)
+	if err == nil {
+		t.Errorf("Expected error but got nil \n")
+	}
+
+	util.CopyFile(tpl, dir+"/two.yaml")
+	tpl1 := filepath.Join(dir, "two.yaml")
+	// force true
+	err = Template(tpl1, val, "", true)
+	if err != nil {
+		t.Errorf("not expecting error but go %s", err.Error())
```

Comment:
can you be more descriptive here - for example: `t.Errorf("error force-generating template (%s)", err)`